tests/common.go

```diff
@@ -1059,6 +1059,11 @@ func ValidatePureSnapshotsSDK(ctx *scheduler.Context, errChan ...*chan error) {
 			processError(err, errChan...)
 		})
 
+		Step("validate Pure local volume paths", func() {
```

Comment:
Why do this check here, instead of at the end once the snapshot is successful? Also does it make sense to move this validation into `AfterEach` ?

Reply:
I suppose that does make sense, yeah